recipes/crisprme/meta.yaml

```diff
@@ -10,12 +10,35 @@ source:
   
 build:
   number: 0
-  noarch: generic
-
+  
 requirements:
   run:
-    - python >3.6
+    - zip
     - crispritz
+    - dash=1.10.0=pyh9f0ad1d_0
+    - dash-bootstrap-components=0.10.0=pyh9f0ad1d_0
+    - dash-core-components=1.9.0=pyh9f0ad1d_0
+    - dash-daq=0.4.0=py_0
+    - dash-html-components=1.0.3=pyh9f0ad1d_0
+    - dash-renderer=1.3.0=pyh9f0ad1d_0
+    - dash-table=4.6.2=pyh9f0ad1d_0
+    - flask=1.1.2=pyh9f0ad1d_0
+    - flask-caching=1.7.1=py_0
+    - flask-compress=1.5.0=pyh9f0ad1d_0
+    - fontconfig=2.13.1=h86ecdb6_1001
+    - freetype=2.10.1=he06d7ca_0
+    - future=0.18.2=py38h32f6830_1
+    - gettext=0.19.8.1=hc5be6a0_1002
+    - gunicorn=20.0.4=py38_0
+    - pip
+    - pip:
```

Comment:
We don't support pip-installed dependencies.

Reply:
Also, is there a reason you're specifying exact packages as dependencies? Even exact versions is a bit restrictive.

Reply:
Hi,
the version in the packages is to be sure any problem is avoided, dash can be very problematic to run without the exact package.
Regarding pip, we will try to check if these dependencies can be satisfied differently.

Thanks

Reply:
Hi, can you review the new file? I've tested it with less restrictive version and no pip packages
thanks